app/schemas/analyze.py

```
@@ -8,7 +8,17 @@
 class AnalyseRequest(BaseModel):
     """Request wrapper for analysis"""
     
-    # example: "How does the authentication system work?"
+from typing import List, Optional
```

Ruff (0.14.5)

Redefinition of unused `List` from line 1

Remove definition: `List`

(F811)